draft-ietf-dnsop-svcb-https.md

```diff
@@ -67,9 +67,9 @@ default server, negotiating a preferred protocol, and providing relevant
 public keys.
 
 For example, when clients need to make a connection to fetch resources
-associated with an HTTPS URI, they currently resolve only A and/or AAAA
+associated with an "https" URI, they currently resolve only A and/or AAAA
```

Comment:
```suggestion
associated with a URI with an "https" or "http" scheme, they currently resolve only A and/or AAAA
```

Reply:
This is an example, and the next part is specific to "https" origins, so I don't know if it makes sense to list both schemes here.

Reply:
The next sentence is
```
This is adequate for services that use
basic HTTP/TLS (fixed port, no QUIC, no {{!ECH=I-D.ietf-tls-esni}}), but
clients that learn additional information can ...
```
Clients don't use HTTP/TLS when the URI has an "http" scheme, so this becomes a non sequitur ... unless we're talking about RFC 8164, which is another layer of complexity that I would prefer not to raise.

This is just supposed to be an example, so there's no need to make it comprehensive.  How about: "For example, when accessing an "https" URI, clients currently resolve only ..."?

Reply:
OK, how about

> For example, HTTP clients currently resolve only A and/or AAAA records for the origin hostname, learning only its IP addresses.  If HTTP clients learn more information about the origin before connecting, they may be able to upgrade "http" URLs to "https", enable HTTP/3 or ECH, or switch to an operationally preferable endpoint.

Reply:
Works for me.

Reply:
Done